ui/appui-react/src/test/frontstage/FrontstageDef.test.tsx

```diff
@@ -187,35 +183,96 @@ describe("FrontstageDef", () => {
     expect(spy).to.be.calledOnceWithExactly("t1");
   });
 
-  it("should activate a widget def", async () => {
-    const def = new FrontstageDef();
-    await def.initializeFromConfig({
-      ...defaultFrontstageConfig,
-      rightPanel: {
-        sections: {
-          start: [
-            {
-              id: "test-widget",
-              defaultState: WidgetState.Hidden,
-            },
-          ],
+  describe("onWidgetStateChangedEvent", () => {
+    it("should open a hidden widget", async () => {
+      const def = new FrontstageDef();
```

Comment:
nitpick: maybe let's use a name that is more specific e.g. `frontstageDef`. Same applies to tests below.

Reply:
Renamed.